touchevents.html

```diff
@@ -130,70 +130,43 @@
   </head>
   <body>
     <section id='sotd'>
-     By publishing this Recommendation, W3C expects that the functionality
-     specified in this Touch Interface Recommendation will not be affected by
-     changes to HTML5 or Web IDL as those specifications proceed to
-     Recommendation.
-
-     The WG has completed and approved this specification's 
-     <a href="http://w3c-test.org/webevents/tests/touch-events-v1/approved/">Test Suite</a> 
-     and created an 
-     <a href="http://www.w3.org/2010/webevents/wiki/TEv1ImplReport">
-     Implementation Report</a> that shows that two or more independent implementations 
-     pass each test.
+      <p>
+        By publishing this Recommendation, W3C expects that the functionality specified in this Touch Interface Recommendation will not be affected by changes to HTML5 or Web IDL as those specifications proceed to Recommendation. The WG has completed and approved this specification's <a href="http://w3c-test.org/webevents/tests/touch-events-v1/approved/">Test Suite</a> and created an <a href="http://www.w3.org/2010/webevents/wiki/TEv1ImplReport"> Implementation Report</a> that shows that two or more independent implementations  pass each test.
```

Comment:
Sorry, this is probably my misunderstanding of part of the "we need to sort out weird tabs/spaces/etc in the markup" discussion we had on list. Maybe it's my personal preference...but I did look at the source for Pointer Events and there we seem to follow (mostly) this pattern as well. Now that it's on GitHub, I can see how it would make diffs more difficult to review here...happy for forced line wrapping to be reintroduced (in a subsequent PR, I'd say, rather than redoing/changing this one further)